client/my-sites/marketing/connections/service-examples.jsx

```diff
@@ -181,6 +184,50 @@ class SharingServiceExamples extends Component {
 			  ];
 	}
 
+	instagram_business() {
+		const label = (
+			<>
+				{ this.props.translate(
+					'Drive engagement and save time by automatically sharing images and video reels to Instagram when you publish blog posts.'
+				) }
+				<div className="instagram-business__requirements">
+					<h4>{ this.props.translate( 'Requirements for connecting Instagram:' ) }</h4>
+					<ol>
+						<li>{ this.props.translate( 'You must have an Instagram Business account.' ) }</li>
+						<li>
+							{ this.props.translate(
+								'Your Instagram Business account must be linked to a Facebook page.'
+							) }
+						</li>
+					</ol>
+				</div>
+				{ this.props.translate(
+					"{{italic}}When you click “connect” you'll be asked to {{strong}}log into Facebook{{/strong}}. If your Instagram Business account isn't listed, ensure it's linked to a Facebook page.{{/italic}}",
+					{
+						components: {
+							strong: <strong />,
+							italic: <i />,
+						},
+					}
+				) }
+				<br />
+				<ExternalLink className="instagram-business__help-link" href="https://wordpress.org">
```

Comment:
Sorry, @manzoorwanijk, there's this one too!

Reply:
Sorry for missing that one. It's done now. Thanks.